src/docs/GetStarted.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="About/Get Started" />
+
+# Get Started
+
+### 1. Install
+
+`yarn add @palmetto/palmetto-components`
+
+or
+
+`npm add @palmetto/palmetto-components`
+
+### 2. Import Global CSS
+
+```
```

Comment:
If we format this as js, it colorizes the line and makes it pop a little more. ```js and ```jsx have the same styling applied. So, either ```js or ```jsx.